ironfish/src/rpc/routes/chain/getNoteWitness.ts

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { ValidationError } from '../../adapters'
+import { ApiNamespace, router } from '../router'
+
+export type GetNoteWitnessRequest = {
+  index: number
+}
+
+export type GetNoteWitnessResponse = {
+  noteSize: number
+  rootHash: string
+  authPathLength: number
```

Comment:
we don't need this since its the length of authPath right?

Reply:
yea probably, not needed I'll remove